tests/model/test_transformers_ulysses.py

```diff
@@ -0,0 +1,206 @@
+import torch
```

Comment:
Add license

Reply:
Do we have a CI to check license?

Reply:
Not yet, I will establish one tomorrow.

Reply:
The license CI is up. Please rebase main